types/hash.js.d.ts

```diff
@@ -0,0 +1,5 @@
+// See https://github.com/indutny/hash.js/issues/18
+declare module 'hash.js/lib/hash/sha/1' {
```

Comment:
There's a builtin `crypto` library: https://developers.cloudflare.com/workers/runtime-apis/web-crypto

Reply:
I tried that at first, but it got a bit complicated - needing to convert the KV value to an ArrayBuffer, and then the result of the crypto method from an ArrayBuffer to a string. I can give it another stab if you prefer that to importing another module though

Reply:
Okay, I switched to `crypto.subtle.digest` 2e46616